libs/platform/src/lib/components/split-menu-button/split-menu-button.component.html

```diff
@@ -1,13 +1,15 @@
-<div class="fd-button-split" role="group" aria-label="button-split">
+<div class="fd-button-split" role="group" aria-label="button-split" [attr.dir]="dir">
     <button
         fd-button
         [attr.id]="id"
         [attr.name]="name"
-        aria-label="buttonLabel"
+        [attr.title]="title"
         [disabled]="disabled"
         [fdType]="type"
         [glyph]="icon"
+        tabindex="0"
         [compact]="contentDensity === 'compact'"
+        aria-label="buttonLabel"
```

Comment:
aria-labe does not make much sense here here you want to use the actual value.

Reply:
for the case of icon button with no text, should hovering over the icon display a tooltip saying what the icon does or as an alt-text for the icon?

Reply:
yes, developer will have option to provide [title], which will be displayed when hover over icon.